basetest.pm

```diff
@@ -166,7 +166,16 @@ sub record_screenmatch {
     if ($foundneedle->has_property('workaround')) {
```

Comment:
Maybe one could just write `if (my $reason = $foundneedle->get_property_value('workaround')) {` instead of running the (linear) search algorithm twice?

Reply:
I modified as your comments.